ui/apps/platform/cypress/integration/compliance-enhanced/ComplianceEnhanced.helpers.js

```diff
@@ -1,14 +1,56 @@
+import { visitFromLeftNavExpandable } from '../../helpers/nav';
 import { visit } from '../../helpers/visit';
 
 // path
 
 export const basePath = '/main/compliance-enhanced';
+export const statusDashboardPath = `${basePath}/status`;
+export const scanConfigsPath = `${basePath}/scan-configs`;
 
-// visit
+// visit helpers
+export const scanConfigsAlias = 'configurations';
+
+const routeMatcherMapToComplianceEnhancedDashboard = null;
```

Comment:
Instead of `null` here for calls below, it seems better to allow as optional argument, only if needed.

* Probably not needed at all for `FromLeftNav` function.
* See comment below for `Dashboard` function.

Reply:
Aha, after reviewing administration events, you can let it stand for `Dashboard` function.

* Small item of notation that elsewhere **For** instead of **To** is the preposition.
* Because these helpers are within compliance-enhanced folder, can we just write Compliance?

That is, `routeMatcherMapForComplianceDashboard`

Reply:
To your first comment, I should add a TODO, because eventually there will be API calls to mock for the new Compliance dashboard, but they don't exist yet. (Brad built it with static data at first, and then several BE resources got diverted to VM for now--so nothing to mock yet.)

For your second comment, yes, I will make those naming changes.